class.jetpack.php

```diff
@@ -487,6 +487,8 @@ private function __construct() {
 			Jetpack_Heartbeat::init();
 		}
 
+		add_filter( 'rest_authentication_errors', array( $this, 'wp_rest_authenticate' ) );
```

Comment:
Is this the best place to handle authentication?  I had thought it was better to use `determine_current_user` for that?